src/pages/index.js

```diff
@@ -62,19 +67,6 @@ profileEditPopup.setEventListeners();
 const imagePopup = new PopupWithimage("#image-modal");
 imagePopup.setEventListeners();
 
```

Comment:
Needs correction: It's also needed to update profile, card and avatar form submit handlers. You should update the info on the server